shared/main.android.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React, {Component} from 'react'
+import hello from './util/hello'
+import {Box, NativeBackAndroid} from './common-adapters/index.native'
+import DumbSheet from './dev/dumb-sheet'
+import {bootstrap} from './actions/config'
+import {connect} from 'react-redux'
+import {listenForNotifications} from './actions/notifications'
+import {getPath} from './route-tree'
+import RenderRoute from './route-tree/render-route'
+import Push from './push/push.native'
+import {setRouteState, navigateUp} from './actions/route-tree'
+import {StatusBar} from 'react-native'
+
+module.hot && module.hot.accept(() => {
+  console.log('accepted update in main.android')
+})
+
+class Main extends Component {
+  constructor (props) {
+    super(props)
+
+    this.props.bootstrap()
+    this.props.listenForNotifications()
+
+    // Introduce ourselves to the service
+    hello(0, 'Android app', [], '0.0.0') // TODO real version
+  }
+
+  componentWillMount () {
+    StatusBar.setTranslucent(true)
```

Comment:
This is new. Added it to match the designs and sync up with iOS (which I needed to add top space for the status bar to since this was previously handled by the Navigator header component).